src/opera_disp_tms/create_tile_map.py

```diff
@@ -41,25 +43,31 @@ def create_bounds_file(info: dict, scale_range: list, output_folder: Path) -> No
         json.dump(extent, outfile)
 
 
-def create_tile_map(output_folder: str, input_rasters: list[str], scale_range: list[float] | None = None) -> None:
+def create_tile_map(measurement_type: str, input_rasters: list[Path]) -> Path:
     """Generate a directory with small .png tiles from a list of rasters in a common projection, following the OSGeo
     Tile Map Service Specification, using gdal2tiles: https://gdal.org/en/latest/programs/gdal2tiles.html
 
     Args:
-        output_folder: Path of the output directory to create
+        measurement_type: Data measurement type to set scale_range and output folder
         input_rasters: List of gdal-compatible raster paths to mosaic
-        scale_range: Optional list of two integers to scale the mosaic by
     """
+
+    output_folder = measurement_type
```

Comment:
It's a bit confusing to have separate `output_folder` and `output_path` variables, I'd suggest making `output_folder` a `Path` object and then just replacing `output_folder` with `str(output_folder)` in the `command` list below.
```suggestion
    output_folder = Path(measurement_type)
```

Reply:
Though I'd probably prefer to see it called `output_dir`.